util/style.py

```diff
@@ -17,3 +19,19 @@
 	"theme": "Primary"
 }
 
+dnd_from_style = {
+	"size_hint": (.3,.1),
+	"background_color": [0,0,0,0],
+	"sources": ['assets/drag-and-drop/shape1.png', 'assets/drag-and-drop/shape2.png', 
+				'assets/drag-and-drop/shape3.png'],
```

Comment:
What changes would be necessary for this to work with images from the asset json?